### PR TITLE
Changing timer name

### PR DESCRIPTION
**Pull request by ChandraAddala**

### CODE REVIEW

File: blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     // todo: these should be instances per client.
     protected final Timer searchTimer = Metrics.timer(getClass(), "Search Duration");
-    protected final Timer getNextTokensTimer = Metrics.timer(getClass(), "getNextTokens Duration");
+    protected final Timer getNextTokensTimer = Metrics.timer(getClass(), "getNextTokens ES metric index query Duration");
```
Comment (VinnyQ) [inline blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java]: NIT:  that's a crazy name, sure you want it like that?  
Also I think the other metric labels are capitalized each word except for minor words.

Comment (VinnyQ) [inline blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java]: I guess there's a few cases that's a long name with lower casing, so nevermind.

**Comment by VinnyQ** [comment]
:+1:

File: blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java (hunk 1 above)
Comment (ChandraAddala) [inline blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java]: How about this? "ES Metric Index Query Duration"

**Pull request merged**